packages/notebook-extension/schema/tracker.json

```diff
@@ -566,6 +566,16 @@
       "keys": ["Ctrl Enter"],
       "selector": ".jp-Notebook.jp-mod-editMode"
     },
+    {
+      "command": "notebook:run-cell",
+      "keys": ["Accel Enter"],
```

Comment:
How about we change the existing Ctrl shortcuts above to Accel instead of leaving them? Right now they are redundant.

Reply:
Never mind. We want *both* Ctrl Enter *and* Accel Enter to work on a mac, so we need to keep the Ctrl Enter shortcuts.

Reply:
Yes both are intended.